orthanc/orthanc-raw/plugin/pixl.py

```diff
@@ -77,6 +77,14 @@ def should_record_headers() -> bool:
     return os.environ.get("ORTHANC_RAW_RECORD_HEADERS", "false").lower() == "true"
 
 
+def should_auto_route():
+    """
+    Checks whether ORTHANC_AUTOROUTE_RAW_TO_ANON environment variable is
+    set to true or false
+    """
+    return os.environ.get("ORTHANC_AUTOROUTE_RAW_TO_ANON", "false").lower() == "true"
+
+
```

Comment:
yeah should change the name to `should_send_to_anon` or something similar. Because we still want the env variable to be able to gate whether the deployment should send to anon